### PR TITLE
NXP-24410: remove deprecated code

### PR DESCRIPTION
**Pull request by driverpt**

### CODE REVIEW

**Comment by kevinleturc** [comment]
👎
This PR can't be reviewed as it. You need to run a T&P (which is in charge of PR creation).

File: nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/batch/Batch.java [hunk 1]
```diff
@@ -247,24 +246,12 @@ public boolean removeFileEntry(String index, TransientStore ts) {
...
-                if (chunkBlobs != null) {
-                    for (Blob blob : chunkBlobs) {
-                        FileUtils.deleteQuietly(blob.getFile().getParentFile());
-                    }
-                }
                 ts.remove(chunkEntryKey);
             }
             fileEntry.beforeRemove();
         }
         // Remove file entry from the store and delete blobs from the file system
```
Comment (kevinleturc) [inline nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/batch/Batch.java]: This comment and the other above is still accurate ?

**Comment by driverpt** [comment]
Closing for T&P

**Comment by driverpt** [comment]
Update: New PR #1754

**Pull request closed without merging**